caluma/caluma_analytics/schema.py

```diff
@@ -181,6 +204,17 @@ class Query:
         filterset_class=CollectionFilterSetFactory(filters.AnalyticsFieldFilterSet),
     )
 
+    pivot_table = graphene.Field(PivotTable, slug=String(required=True))
```

Comment:
`class Mutation` is missing `save_` and `remove_` properties for bot `pivot_table` and `aggregate_field`

Doing this will require to also define `PivotTableSerializer` and `AggreateFieldSerializer`

Reply:
Moved both types together into the same model / gql mutations